aten/src/ATen/native/native_functions.yaml

```diff
@@ -10223,6 +10220,14 @@
   dispatch:
     CPU, CUDA: linalg_lstsq_out
 
+- func: linalg_matrix_exp(Tensor self) -> Tensor
+  python_module: linalg
+  variants: function
+  dispatch:
+    CPU, CUDA: linalg_matrix_exp
+
+- func: linalg_matrix_exp_backward(Tensor self, Tensor grad) -> Tensor
```

Comment:
Why is the backward exposed in native_functions.yaml?

Reply:
It was there already, this is just renaming and moving around. This is moved to `FunctionsManual.cpp`  in the second PR of this stack.

Reply:
I think matrix_exp_backward is actually used. Can we do the following:

- leave matrix_exp_backward as-is
- not expose a linalg_matrix_exp_backward